transformations/gender_randomizer/README.md

```diff
@@ -0,0 +1,45 @@
+# Gender Randomizer 🦎  + ⌨️ → 🐍
+
+Author name: Tabitha Sugumar
+Author email: t.sugumar@elsevier.com
+Author Affiliation: Elsevier
+
+## What type of a transformation is this?
+This transformation changes names in English texts, randomizing selection so that there's an even chance of male and female names. Acknowledging that names are not deterministic identifiers of someone's gender/pronouns, here pronouns are modified to she/her/herself if the selected name is in names/female.txt and to he/his/him/himself if the selected name is in names/male.txt. These files can be modified or replaced as desired.
+
+## What tasks does it intend to benefit?
+This is intended to avoid gender bias in natural language processing models. Run this transformation on text data prior to using it to train a model.
+
+## Previous Work
```

Comment:
Importantly please add a Data and Code Provenance section to your transformation. Also, seems you've added about a 109 files which are hard to evaluate. I would suggest moving this into a separate pip project out of this and then adding it to the requirements.txt.

Reply:
Thanks! I've expanded on the data and code provenance, and put the description in a Data and Code Provenance section in the Readme.

On the 109 files -- most of them come from the coreferee directory -- this actually already exists as a library installable by pip, but when I was working on this was only installable in python 3.8 and the current version requires python 3.9. Since these transformations are required to be compatible with python 3.7, I downloaded here to make it installable in python 3.7.